audobject/core/object.py

```diff
@@ -167,12 +167,14 @@ def id(self) -> str:
     @staticmethod
     def from_dict(
             d: typing.Dict[str, typing.Any],
+            root: str = None,
             **kwargs,
     ) -> 'Object':
         r"""Create object from dictionary.
 
         Args:
             d: dictionary with variables
+            root: if dictionary is read from a file, set to source directory
```

Comment:
As the dictionary is already in `d` and we don't read it here, maybe its better to understand if we change to 
```suggestion
            root: if dictionary was read from a file, set to source directory
```

Reply:
Makes sense, I changed it accordingly.